src/components/dashboard/components/dictionary/common/DictionaryModal.jsx

```diff
@@ -253,7 +251,6 @@ export class DictionaryModal extends React.Component {
                       onChange={this.onChange}
                       value={data.public_access}
                     >
-                      <option value="" />
```

Comment:
Has this change anything to do with the ticket?

Reply:
Yeah, it has changed as it was initially empty so I removed it to give the public access a placeholder.